spec/controllers/accounts_controller_spec.rb

```diff
@@ -120,6 +120,11 @@
           expect(response.body).to include(I18n.t('stream_entries.pinned'))
         end
 
+        it 'does not render private pinned status' do
```

Comment:
This spec _should_ pass but something about my local environment was making controller specs raise errors. Something about Webpack not generating a manifest. I'll look into this in the future, but I wanted to make sure this was in here.